Block/Checkout/Onepage/Success/OfflineAdditionalInformation.php

```diff
@@ -0,0 +1,86 @@
+<?php
```

Comment:
According to the structure of Magento itself and its documents, I think they are strongly suggesting and encouraging developers to apply SOLID principle to the code. By that, I think the previous implementation that separate the concern between each of payment method's additional information has already settled a foundation of SOLID principle. 

By grouping them into one file, we are introducing lots of if-else conditional block into many part of the code (at least 2-3 parts for now). Hence, we are repeating the giant `APMBuilder` class by keep adding more condition when a new payment method arrives.

What if we just leave it stay as separate classes and maybe have one parent class for those repeated method to be grouped and be able to re-use in every of sub-classes?

(Just wrote some idea out https://gist.github.com/guzzilar/d66e30170fca85c8a6c2026850204476)

Reply:
(I think reducing and merging the repeated code is a really great idea. Just, if the merged-code needed to be conditioned with if-else, then probably the code is not really fit to be in the same place.)

Reply:
Thanks @guzzilar for your review, I have updated code according to your suggestion. Please help to review this again.